resources/assets/uad_lists.json

```
@@ -15947,6 +15947,15 @@
     "labels": [],
     "removal": "Recommended"
   },
+  {
+    "id": "com.mi.globalminusscreen",
+    "list": "Oem",
+    "description": "Xiaomi App vault\nhttps://play.google.com/store/apps/details?id=com.mi.android.globalminusscreen",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
```

```suggestion
    "removal": "Advanced"
```
Don't think it's harmful to the point it should be `Recommended`. I think `Advanced` fits better, because some users may want to use it.